framework/include/utils/HasMembers.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
Need a header here

Reply:
I noticed a lot of missing headers. Is there a heading pre-check?

Reply:
There isn't. And whoops, there are quite a few more.